hydra-node/test/Hydra/Chain/Direct/Contract/Close.hs

```diff
@@ -209,6 +211,8 @@ genCloseMutation (tx, _utxo) =
                 (replacePolicyIdWith Fixture.testPolicyId otherHeadId healthyOpenHeadTxOut)
                 (Just $ toScriptData healthyCloseDatum)
             ]
+    , SomeMutation (Just "burning is forbidden") MutateTokenBurning
+        <$> changeMintedTokens tx (valueFromList [(AssetId Fixture.testPolicyId "badTokenBurned", Quantity (-1))])
```

Comment:
Should: not only test burning, but also minting should not be allowed. In fact, any minting (not only of our policy). So we could generate maybe three cases:
* Burn a single token of our policy (what you did here)
* Mint a single token of our policy
* Burn or mint an `arbitrary` token of an `arbitrary` policy

This (generator) should cover the interesting edge cases with high enough frequency to get us convinced this actually works (only allows keeping `mint = 0`)